### PR TITLE
#6 updates README, more inline with requirements

### PR DESCRIPTION
**Pull request by cadye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,24 +23,16 @@ Carl: Gradle.
...
-
-### Execution
+Created an Email class in Java with the package javax.mail. This Email class handles the notifications. For every push, it will send the CI result to every participant in the group project. The CI results include if the committed code compiled successfully and it passed all tests. The Email class will return if the is tested by checking if the email was successfully sent.
```
Comment (ElisabethChen) [inline README.md]: I have also changed the README file so it would be good if you can also add the changed part.

**Review by ElisabethChen — CHANGES_REQUESTED** [review]
It looks good to me! Just need to add the changes that I added to notification part.

**Review by ElisabethChen — APPROVED** [review]
Looks good to me!

**Review by YaourtNature83 — APPROVED** [review]
Good

**Review by hemenali11 — APPROVED** [review]

**Pull request merged**